### DISCUSSION TITLE
Is there any keyboard shortcut to switch between these tabs?

### DISCUSSION THREAD

**Discussion by why-not-try-calmer** [community/community#49401] [Pull Requests] [answered]
### Select Topic Area

Question

### Body

![Capture d’écran du 2023-03-07 09-18-27](https://user-images.githubusercontent.com/19554213/223385748-1783556e-111f-46ae-b33e-477005a9bdde.png)

That would greatly help my tentatively keyboard-centric workflow

**Comment by maahaMaya** [comment, marked as answer]
[https://docs.github.com/en/get-started/using-github/keyboard-shortcuts]((https://docs.github.com/en/get-started/using-github/keyboard-shortcuts))

all the shortcuts are mentioned here.

**Reply by why-not-try-calmer** [reply]
It's not listed. What's the best place to make a featurer request?

**Reply by queenofcorgis** [reply]
@maahaMaya thanks for sharing the documentation - @why-not-try-calmer you're in the right place to make a feature request - feel free to update your discussion for your use case and any other details you feel are relevant.

**Reply by why-not-try-calmer** [reply]
> @maahaMaya thanks for sharing the documentation - @why-not-try-calmer you're in the right place to make a feature request - feel free to update your discussion for your use case and any other details you feel are relevant.

Thanks, so how do you recommend I go about officially turning this into a feature request?

**Reply by queenofcorgis** [reply]
To make a feature request, I'd update your original post here with details of what you would like and your use cases so our team can properly evaluate your request. Once that's done let me know and I'll tag it as feedback!

**Reply by why-not-try-calmer** [reply]
Unfortunately I cannot edit my post but here is the feature request:

![image](https://github.com/community/community/assets/19554213/1b77e66c-f7b6-44c0-8aaa-82b78cf526dd)

_These tabs (available under a Pull Request) are currently not switchable through shortcuts. It would be nice, I think, if they were. I'd suggest chording `t` with `[1-4]`, for lack of a better alternative, following the logic used for switching between tabs in the main navigation area (`g` chording with a letter)._

**Reply by queenofcorgis** [reply]
@why-not-try-calmer ah thank you so much for this explanation and your suggestion on how it would work. I've tagged this as product feedback, we can't promise anything (in full transparency), but our team will review this.

**Reply by alanmoo** [reply]
I've been trying to find a solution for this for years as well; Github offers an amazing variety of keyboard shortcuts but they're much more useful when you can connect from one view to another using just the keyboard.

**Comment by marry231** [comment]
GitHub offers a comprehensive list of keyboard shortcuts to help users navigate and perform tasks more efficiently. These shortcuts are especially useful for developers who spend a lot of time on the platform and want to speed up their workflow. Whether you're managing issues, reviewing pull requests, or browsing code repositories, mastering these shortcuts can save significant time. GitHub’s official documentation provides a detailed guide to all available shortcuts, categorized by activity and page context. You can explore the full list by visiting the official page: [GitHub Keyboard Shortcuts](https://docs.github.com/en/get-started/using-github/keyboard-shortcuts).

Interestingly, just as developers benefit from efficiency in their coding environment, musicians can also benefit from online tools that streamline their learning process. For example, ukulele players looking to learn Huling El Bimbo by Eraserheads can find helpful chords and tabs on this [ukulele tab page](https://www.ukulele-tabs.com/uke-songs/eraserheads/huling-el-bimbo-uke-tab-26901.html).

**Comment by florajrose-web** [comment]
Thanks for sharing this! Keyboard shortcuts really boost workflow efficiency, especially for developers toggling between tabs a lot.

On a related note — if you’re curious about how control systems work under‑the‑hood in automation (and how those same design patterns can influence UI/UX flows like shortcut handling), this article is a good read:[ 🔗 ](https://nedes.us/what-are-the-control-systems-used-in-automation/)

I found it clears up how feedback loops and state transitions are engineered — useful when you’re thinking of UX patterns and productivity tools.
